xgi/classes/hypergraph.py

```diff
@@ -964,3 +964,47 @@ def dual(self):
         dual._hypergraph = deepcopy(self._hypergraph)
 
         return dual
+
+    def cleanup(
+        self,
+        remove_isolates=True,
+        remove_singletons=True,
+        remove_multiedges=True,
+        reindex=True,
+    ):
+        """Removes potentially undesirable artifacts from the hypergraph.
+
+        Parameters
+        ----------
+        remove_isolates : bool, optional
+            Whether to remove isolated nodes, by default True
+        remove_singletons : bool, optional
+            Whether to remove singleton edges, by default True
+        remove_multiedges : bool, optional
+            Whether to remove duplicate edges, by default True
+        reindex : bool, optional
```

Comment:
the name of the function contains the word `label` so to keep it consistent, this parameter should be called `relabel` too

Reply:
Done.